src/twisted/runner/procmon.py

```diff
@@ -145,11 +175,16 @@ class ProcessMonitor(service.Service):
     @ivar _reactor: A provider of L{IReactorProcess} and L{IReactorTime}
         which will be used to spawn processes and register delayed calls.
 
+    @type log: C{Logger}
+    @ivar log: An instance of twisted.logger.Logger used for propagating
```

Comment:
Since that is given in the type (and the `L{}` syntax will create a link in the API docs), this should be "The logger used to propagate log messages from spawned processes" or so.